### PR TITLE
add gov.br

### PR DESCRIPTION
**Pull request by rafaelvannucci**
Add Brazilian government unified services portal

### CODE REVIEW

File: quirks/password-rules.json [hunk 1]
```diff
@@ -102,6 +102,10 @@
     "girlscouts.org": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789$#!];"
     },
+    "gov.br": {
+        "password-rules": "minlength: 8; maxlength: 70; required: [!"#$%&()*+,-./:;<=>?@[\]^{|}~.];"
```
Comment (rmondello) [inline quirks/password-rules.json:106]: I don’t think you can include a quote here like this. This needs to be escaped, somehow.

File: quirks/password-rules.json [hunk 2]
```diff
@@ -102,6 +102,10 @@
...
+        "exact-domain-match-only": true
```
Comment (rmondello) [inline quirks/password-rules.json:107]: So you’re trying to say that no subdomains of gov.br should have these rules?

Comment (rafaelvannucci) [reply quirks/password-rules.json]: edited to match actua rules.

File: quirks/password-rules.json (hunk 1 above)
Comment (rafaelvannucci) [reply quirks/password-rules.json]: edited to match actua rules.

**Comment by rmondello** [comment]
> `minlength: 8; maxlength: 70;`

If that’s the websites requirements, it’s one of the most flexible websites I’ve ever seen! I’m not sure what value there is in tracking a maxlength of 70 for this website, but I’d be happy to hear arguments about this!

**Comment by rafaelvannucci** [comment]
> > `minlength: 8; maxlength: 70;`
> 
> If that’s the websites requirements, it’s one of the most flexible websites I’ve ever seen! I’m not sure what value there is in tracking a maxlength of 70 for this website, but I’d be happy to hear arguments about this!

You're right @rmondello :/ It's very flexible. I'll go ahead and close this PR, as I don't have any argument about that.

**Pull request closed without merging**